### DISCUSSION TITLE
raise `Kafka::PolicyViolation` error when creating topic

### DISCUSSION THREAD

**Discussion by xiaoronglv** [zendesk/ruby-kafka#911] [Q&A]
I have created a Kafka cluster in the confluent cloud. the cluster is in the `us-west-2` of AWs.

![2021-08-02_14-13-00](https://user-images.githubusercontent.com/1224077/127812490-d975fc2e-b2c8-48b7-ac9c-cfcadbf7ea09.jpg)

Then I use ruby-kafka to create a new topic.

![2021-08-02_14-11-09](https://user-images.githubusercontent.com/1224077/127812296-af9f5092-9abd-4bdb-9fde-0ebb60b034c4.jpg)

it raises `Kafka::PolicyViolation` error. what does that mean? how to fix it?

is it caused by compatibility or permission?

**Comment by kylejw2** [comment]
Any updates on this? @xiaoronglv Were you able to determine why this error was being thrown?

**Reply by xiaoronglv** [reply]
Nope, I haven't figured it out.

I used a stupid way to create the topic:

1. go to confluent's dashboard
2. create topics through their UI.

![image](https://user-images.githubusercontent.com/1224077/179131076-80ccfad3-9208-4306-931b-abe1a55efc9f.png)

**Reply by kylejw2** [reply]
The error might be a confluent limitation.  You'll need a dedicated cluster in order to create topics programmatically. The basic setup doesn't allow this
